modules/business-intelligence/pages/create-stunning-visualizations-in-power-bi-using-data-from-teradata-vantage.adoc

```diff
@@ -131,4 +131,7 @@ You can combine data from many sources with Power BI Desktop. Look at the follow
 * https://docs.microsoft.com/en-us/power-bi/desktop-data-sources[Data Sources in Power BI Desktop]
 * https://docs.microsoft.com/en-us/power-bi/desktop-shape-and-combine-data[Shape and Combine Data with Power BI Desktop]
 * https://docs.microsoft.com/en-us/power-bi/desktop-connect-excel[Connect to Excel workbooks in Power BI Desktop]
-* https://docs.microsoft.com/en-us/power-bi/desktop-enter-data-directly-into-desktop[Enter data directly into Power BI Desktop]
\ No newline at end of file
+* https://docs.microsoft.com/en-us/power-bi/desktop-enter-data-directly-into-desktop[Enter data directly into Power BI Desktop]
+
+== For any questions:
```

Comment:
The current convention is that we use section titles to announce what a section will be about. We don't include prompts in the section titles.

You could change the section title to something like this:

```suggestion
== Getting help
```